source/expressions.tex

```diff
@@ -3274,8 +3274,8 @@
 
 \item If \tcode{E2} is a non-static data member and the type of
 \tcode{E1} is ``\cvqual{cq1 vq1} \tcode{X}'', and the type of \tcode{E2}
-is ``\cvqual{cq2 vq2} \tcode{T}'', the expression designates the named
-member of the object designated by the first expression. If \tcode{E1}
+is ``\cvqual{cq2 vq2} \tcode{T}'', the expression designates the associated
```

Comment:
"corresponding" would be a better word than "associated" here, to indicate we mean the member subobject corresponding to `E2`.

Reply:
I agree. The relationship is a bijection, commonly called a "one-to-one correspondence". "Corresponding" is better. Changed.